Exercise/compute_lowest_terms.py

```diff
@@ -4,53 +4,51 @@ def lowest_terms(x):
 	numbers = x.split("/")
```

Comment:
Function missing docstring. See distance_to_destination function in Example/ folder for sample docstring